src/tasks/deploy.mk

```diff
@@ -85,6 +89,23 @@ deplo%: ## deploy: deploy the app to heroku
 	heroku pipelines:promote -a $(HEROKU_APP_STAGING)
 	heroku dyno:scale web=0 -a $(HEROKU_APP_STAGING)
 
+	$(MAKE) change-api
+
+change-api:
+	curl \
+  		--header "Content-Type: application/json" \
+  		--header "x-api-key: $(CHANGE_API_KEY)" \
+		--request POST \
+  		--data "{ \
+			\"user\": { \
+				\"githubName\": \"$(CIRCLE_USERNAME)\" \
+			}, \
+			\"environment\": \"production\", \
+			\"systemCode\": \"$(VAULT_NAME)\", \
```

Comment:
We know this isn't 100% correct, e.g. next-esinterface vs next-es-interface.

I _think_ it'd be good for us to update the systemcodes where this doesn't work, rather than try and get systemCodes into the CircleCI environment variables.

Reply:
Only thing is that teams with conflicting `VAULT_NAME` vs `systemCode` will have to create a completely new entry in biz-ops as `systemcodes` cannot be changed when created. Or they'll need to rename their github repo/`VAULT_NAME`.

Are we fine with that?